.github/workflows/backend.yml

```diff
@@ -221,3 +221,19 @@ jobs:
           m2-cache-key: be-check
       - name: Check namespaces
         run: clojure -M:ee:drivers:check
+
+  be-check-stub:
+    needs: files-changed
+    if: |
+      always() &&
+      github.event.pull_request.draft == false &&
+      needs.files-changed.outputs.backend_all == 'false'
```

Comment:
For the easier reference, the original tests had this logic:
`if: github.event.pull_request.draft == false && needs.files-changed.outputs.backend_all == 'true'`